### PR TITLE
Observation new platform geofilter

### PR DESCRIPTION
**Pull request by teamdraftbox**
Geo filter added to  observation stats aggregation 
Bulk Upload minor bug fixes
Datatable Name and Dataset Name filter added to observation list page

### CODE REVIEW

File: src/main/java/com/strandls/observation/es/util/ObservationESDocument.java [hunk 1]
```diff
@@ -105,6 +105,7 @@
 	private String location_scale;
 	private Long dataset_id;
 	private String dataset_title;
+	private String data_table_title;
```
Comment (akrudra) [inline src/main/java/com/strandls/observation/es/util/ObservationESDocument.java:108]: In query data_table_id, data_table_title, both are added, Here data_table_id is missing ??? why ?

File: src/main/java/com/strandls/observation/es/util/ObservationListPageMapper.java [hunk 2]
```diff
@@ -458,6 +461,7 @@ public ObservationListPageMapper() {
 	 * @param userGroup
 	 * @param customField
 	 * @param tags
+	 * @param placeName
```
Comment (akrudra) [inline src/main/java/com/strandls/observation/es/util/ObservationListPageMapper.java:464]: remove this unnecesaary comments, Place name is not present as the property of the class

File: src/main/java/com/strandls/observation/es/util/ConstructESDocument.java [hunk 3]
```diff
@@ -39,9 +39,12 @@
 				+ "place_name, reverse_geocoded_name, flag_count, geo_privacy, last_revised,  "
 				+ "visit_count, is_checklist, to_date,  " + "checklist_annotations, "
 				+ "is_locked, language_id, location_scale,  "
-				+ "dataset_id, dataset_title, repr_image_id, repr_image_url, protocol, no_of_images, no_of_videos, no_of_audio, "
+				+ "dataset_id, dataset_title,data_table_id,data_table_title, repr_image_id, repr_image_url, protocol, no_of_images, no_of_videos, no_of_audio, "
```
Comment (akrudra) [inline src/main/java/com/strandls/observation/es/util/ConstructESDocument.java:42]: Query is changed, Did you made the same change the main query.? Please update that as well

**Review by akrudra — CHANGES_REQUESTED** [review]

File: src/main/java/com/strandls/observation/es/util/ConstructESDocument.java (hunk 3 above)
Comment (teamdraftbox) [reply src/main/java/com/strandls/observation/es/util/ConstructESDocument.java:42]: yes, and i have updated the ng-98 ticket as well

File: src/main/java/com/strandls/observation/es/util/ObservationESDocument.java (hunk 1 above)
Comment (teamdraftbox) [reply src/main/java/com/strandls/observation/es/util/ObservationESDocument.java:108]: data_table_id has already been added

https://github.com/strandls/biodiv-observation/blob/f8cd18dc785051ae9572dcd193ace830815409d8/src/main/java/com/strandls/observation/es/util/ObservationESDocument.java#L117

File: src/main/java/com/strandls/observation/es/util/ConstructESDocument.java (hunk 3 above)
Comment (teamdraftbox) [reply src/main/java/com/strandls/observation/es/util/ConstructESDocument.java:42]: Screenshot from 2021-08-12 15-31-38

**Review by akrudra — APPROVED** [review]
Seems good to merge

**Pull request merged**